mode/smalltalk/smalltalk.js

```diff
@@ -91,6 +92,7 @@ CodeMirror.defineMode('smalltalk', function(config) {
     } else if (/[\w_]/.test(aChar)) {
       stream.eatWhile(/[\w\d_]/);
       token.name = state.expectVariable ? (keywords.test(stream.current()) ? 'keyword' : 'variable') : null;
+      token.name = state.expectVariable ? (variables.test(stream.current()) ? 'variable-2' : 'variable') : null;
```

Comment:
Could probably be better represented like,
`    } else if (/[\w_]/.test(aChar)) {
      stream.eatWhile(/[\w\d_]/);
	  if(keywords.test(stream.current()){ 
	  token.name = 'keyword'
	  } else if(variables.test(stream.current){
		  token.name = 'variable-2'
	  } else { token.name = 'variable' }`